content/logs/aws.md

```diff
@@ -0,0 +1,122 @@
+---
+title: AWS Log Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+## Overview
+
+Push your logs from ELB, S3, CloudTrail, VPC, CloudFront and CloudWatch into an S3 bucket bind to a lambda function. 
```

Comment:
Should we preface any amazon services with aws or amazon as they require?

Reply:
Push your logs from ELB, S3, CloudTrail, VPC, CloudFront, and CloudWatch into an S3 bucket bound to a Lambda function.

But is the Lambda bound to the S3 bucket? or does the Lambda push the logs to that bucket?

Reply:
Change the paragraph to:

Push your AWS log information to Datadog using a Lambda function bound to an S3 bucket. First, configure your AWS services to push logs to S3. A Lambda function is then triggered and processes the log file, eliminating the need for additional services to poll for that information.

Reply:
👍